src/main/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowBranchProjectFactory.java

```diff
@@ -82,6 +98,17 @@ public boolean equals(Object obj) {
 
     @Extension public static class DescriptorImpl extends AbstractWorkflowBranchProjectFactoryDescriptor {
 
+        private String scriptPath = SCRIPT;
+
+        public DescriptorImpl(){ this.scriptPath = SCRIPT; }
+
+        @DataBoundConstructor
```

Comment:
Again, thanks for the hint. I am assuming that the descriptor does not actually need to have any references to `scriptPath` - don't know if that is correct. I could not make any sense of the resources that I was able to find about `hudson.model.Descriptor` - at least regarding this case.